a10_octavia/controller/worker/tasks/cert_tasks.py

```diff
@@ -230,14 +239,22 @@ def execute(self, cert_data, vthunder):
                                                                  passphrase=cert_data.key_pass)
 
                 try:
-                    if 'cert' in old['client-ssl'].keys() and 'key' in old['client-ssl'].keys():
-                        if old['client-ssl']['cert'] != cert_data.cert_filename:
-                            self.axapi_client.file.ssl_cert.delete(
-                                private_key=old['client-ssl']['key'],
-                                cert_name=old['client-ssl']['cert'])
-                        if old['client-ssl']['key'] != cert_data.key_filename:
-                            self.axapi_client.file.ssl_key.delete(
-                                private_key=old['client-ssl']['key'])
+                    if utils.acos_version_cmp(vthunder.acos_version, "5.2.1") >= 0:
+                        old_certificate_list = old['client-ssl']['certificate-list'][0]
+                        if 'cert' in old_certificate_list and 'key' in old_certificate_list:
+                            if old_certificate_list['cert'] != cert_data.cert_filename:
+                                self.axapi_client.file.ssl_cert.delete(
```

Comment:
So,
- line 236 `self.axapi_client.slb.template.client_ssl.update` we will configure both cert/keys in client-ssl
- and here we will remove the old cert/key files and **the old cert/key will be removed from client-ssl automatically**?